TAU release 1.1.6

TomaszLukawskiSam opened the pull request:
Dedicated platform: Tizen 5.5

New features in 1.1.0:

- new widgets: Graph, Dimmer
- Keyboard support
- TV landscape layout

Changes in version 1.1.6:
- (Mobile)(UIComponents) added dividers filtering for IndexScrollbar example
- (Mobile)(SectionChanger) Make SectionChanger resizable
- (Mobile)(Highlight) Fix for two items highlights in the same time
- (Mobile)(Drawer) Fix for filling colored list in drawer widget
- (Mobile)(Page) Storing and restoring page scroll position on page change
- (Mobile)(UIComponents) Fix for duplicate page ID in bottom button example
- (Wearable) Number, Time and Date pickers: remove "set" button on widget destroy
- (Mobile) Expandalbe list: apply active state only for it's header
- (Wearable) Change positioning of swipelist item text and icon
- (Wearable) Fix selector icons disappearing
- (Wearable)(UIComponents) Add missing btn-text class for button
- (Wearable)(UIComponents) Added marquee widget to title in graph examples
- (util) globalize: block data loading when data has already exists
- (examples) UIComponents: Duplicated ID of elements causes change page issue

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

lslachciak reviewed (changes requested):
issue with back navigation from recommendations->textinputwith button
holding release until fix is done

TomaszLukawskiSam committed:
TAU release 1.1.6

Dedicated platform: Tizen 5.5

New features in 1.1.0:

- new widgets: Graph, Dimmer
- Keyboard support
- TV landscape layout

Changes in version 1.1.6:

- (Mobile)(UIComponents) added dividers filtering for IndexScrollbar example
- (Mobile)(SectionChanger) Make SectionChanger resizable
- (Mobile)(Highlight) Fix for two items highlights in the same time
- (Mobile)(Drawer) Fix for filling colored list in drawer widget
- (Mobile)(Page) Storing and restoring page scroll position on page change
- (Mobile)(UIComponents) Fix for duplicate page ID in bottom button example
- (Wearable) Number, Time and Date pickers: remove "set" button on widget destroy
- (Mobile) Expandalbe list: apply active state only for it's header
- (Wearable) Change positioning of swipelist item text and icon
- (Wearable) Fix selector icons disappearing
- (Wearable)(UIComponents) Add missing btn-text class for button
- (Wearable)(UIComponents) Added marquee widget to title in graph examples
- (util) globalize: block data loading when data has already exists
- (examples) UIComponents: Duplicated ID of elements causes change page issue

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

TomaszLukawskiSam commented:
> issue with back navigation from recommendations->textinputwith button
> holding release until fix is done

The issue is not related to this release, but found and fixed one issue with duplicate IDs for page elements.
